main/src/cgeo/geocaching/AbstractDialogFragment.java

```diff
@@ -227,7 +227,10 @@ protected final void addCacheDetails() {
         }
 
         // favorite count
-        details.add(R.string.cache_favorite, cache.getFavoritePoints() + "×");
+        final int favCount = cache.getFavoritePoints();
+        if (favCount >= 0) {
+            details.add(R.string.cache_favorite, favCount + "×");
```

Comment:
The `"×"` should be part of the resource string.

Reply:
This is [intended] to be the only place where "x" is displayed. Since it doesn't have neither translation nor smart logic, I think we can keep it in this only place and should be fine.

Reply:
You can use `main/res/values/strings_not_translatable.xml` for strings that are not intended for translation.

Reply:
It's just one/two numbers. There is no any smartness in formatting. It was not in resources before for years. Maybe it can be added, but it does not belong to this change anyway.

Reply:
Appending the `×` here and in `CacheDetailActivity.java` should be consistent but currently the code fragments are not connected. That's why introducing a resource improves the code quality.

The resource issue will come up again in #7441 anyway, so starting here would be helpful.
In general, it is always a good thing to clean up the code you are currently changing.